### PR TITLE
[WIP] Issue #942: Add Custom VM for Projects

### PR DESCRIPTION
**Pull request by lhridley**
Added CustomVM for Projects page

-- This page does not show up in the sidebar from other pages in the project.  When you navigate to another subpage this page disappears from the sidebar menu -- it's only visible from the main page of the project.

Issue #942

### CODE REVIEW

**Comment by kostajh** [comment]
@lhridley per the "in house additions" to the [git commit messages](http://developers.savaslabs.com/en/latest/git.html#git-commit-messages) docs, could you please rewrite the git commit as `Issue #942: Add Custom VM for Projects page`

File: custom-vm.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Custom VM
+
+There are times when we are developing a site that will be hosted in an environment that requires us to
+create a custom VM in order to more closely approximate the production environment where the site will
+eventually be hosted.  In this case we can create a VM box for that project to use instead of Drupal VM.
+
+## Hosting
+
+Custom VMs are hosted on the Savas Labs Atlas Cloud (savas) account at https://atlas.hashicorp.com
```
Comment (chrisarusso) [inline custom-vm.md]: @lhridley let's link this with markdown syntax.

File: custom-vm.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Custom VM
+
```
Comment (chrisarusso) [inline custom-vm.md]: @lhridley here can we add some specifics about base box that you created and where it lives?

**Comment by kostajh** [comment]
I'm going to close this since things have changed quite a bit since it was filed. Sorry for the delay in reviewing!

**Pull request closed without merging**